src/main/java/com/redhat/devtools/lsp4ij/internal/SupportedFeatures.java

```diff
@@ -73,15 +73,18 @@ public class SupportedFeatures {
         signatureHelpCapabilities.setSignatureInformation(signatureInformationCapabilities);
         textDocumentClientCapabilities.setSignatureHelp(signatureHelpCapabilities);
 
+        // Definition support
+        final var declarationCapabilities = new DeclarationCapabilities();
+        textDocumentClientCapabilities.setDeclaration(declarationCapabilities);
+
         // Definition support
         final var definitionCapabilities = new DefinitionCapabilities();
         definitionCapabilities.setLinkSupport(Boolean.TRUE);
         textDocumentClientCapabilities.setDefinition(definitionCapabilities);
 
-        // TODO support type definition
-        //final var typeDefinitionCapabilities = new TypeDefinitionCapabilities();
-        //typeDefinitionCapabilities.setLinkSupport(Boolean.TRUE);

```

Comment:
what's link support? Can we support it?

Reply:
We can set as supported but I don't use information about LocationLink which is used generally to higlight with underline the link in the editor.

In Find Usages case, I don't use it, but I take care of LocationLink if server send that. 

I will mark as supported even if we don't use origin info.